plugins/modules/xbps.py

```diff
@@ -67,6 +67,26 @@
         type: bool
         default: true
         version_added: '0.2.0'
+    root:
+        description:
+            - The full path for the target root directory.
+        type: str
+        version_added: '10.1.0'
+    repositories:
+        description:
+            - Repository URL(s) to prepend to the repository list for the
+              package installation.
+              The URL can be a URL to a repository for
+              remote repositories or a path for local repositories.
+        type: list
+        elements: str
+        version_added: '10.1.0'
+    accept_pubkey:
+        description:
+            - Wether or not repository signing keys should be automatically accepted.
```

Comment:
```suggestion
            - Whether or not repository signing keys should be automatically accepted.
```
(TIL that `wether` is also an [existing word](https://en.wikipedia.org/wiki/Wether), but definitely not what you meant ;) )